### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by jasperdanan**
Merge?

### CODE REVIEW

File: raspicheck.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import discord
+from discord.ext import commands
+from __main__ import set_cog, send_cmd_help, settings
+import asyncio
+import psutil
```
Comment (Chovin) [inline raspicheck.py]: I believe this needs to be pip-installed. There's a new way to do so:
image

Comment (Chovin) [reply raspicheck.py]: actually, the new, new way to do it would be to just import it and put it in the cog's info.json requirements

**Comment by Chovin** [comment]
cog needs to be in its own folder with an info.json

please see this for a guide: https://twentysix26.github.io/Red-Docs/red_cog_how_to_publish/
here is an example: https://github.com/Kowlin/refactored-cogs/tree/master/sentry

#16 #18 also

**Comment by jasperdanan** [comment]
added info.json file, merge?

**Pull request merged**